addon/components/o-s-s/information-section.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  data-control-name="information-section"
+  class="oss--information-section border-radius-md border-color-gray-200 box-shadow-md"
+  ...attributes
+>
+  <div
+    class="fx-row fx-malign-space-between fx-xalign-center padding-px-18 {{this.plainClass}}"
```

Comment:
if we are going to have a getter in it, why not just do a `computedClasses` getter that have all of them + the conditional ones and `Array#join` them?

Reply:
Didn't see the need to array join but the full string is returned in a headerComputedClasses getter